ydb/docs/en/core/concepts/_includes/ttl.md

```diff
@@ -176,6 +190,45 @@ The example below shows how to use the `modified_at` column with a numeric type
 
 {% endlist %}
 
+### Enabling eviction to Object Storage for an existing table {#enable-tiering-on-existing-tables}
+
+{% note info %}
+
+This functionality is only available for column-oriented tables. For row-oriented tables, this functionality is under development.
```

Comment:
We need to create a standardized include for such notes similar to https://github.com/ydb-platform/ydb/blob/main/ydb/docs/en/core/_includes/not_allow_for_olap_note.md (but opposite). Otherwise, such places will all look differently and be harder to find later.